src/assets/js/main.js

```diff
@@ -187,27 +77,158 @@ function initVideoModal() {
     }
 }
 
-$(document).ready(function () {
-        function createGallery() {
-            for (var i = 0; i < arguments.length; i++) {
-                const galleryName = arguments[i];
-                const selector = $('#' + galleryName + ' .selector li');
-                selector.hover(function () {
-                    switchBanner.call(this, galleryName);
-                });
-
-                selector.focus(function () {
-                    switchBanner.call(this, galleryName);
-                });
-            }
-        }
-
-        createGallery(
-            'galleryOne',
-            'galleryTwo',
-            'galleryThree',
-            'galleryFour',
-            'galleryFive',
-            'gallerySix');
+function fixNav() {
+  var t = $(document).scrollTop(),
+    f = $("#page-footer").offset().top,
+    hh = $("#page-header").height(),
+    banner = $(".banner"),
+    bb = banner.length > 0 ? banner[0].getBoundingClientRect().bottom : hh,
+    headerHeight = Math.max(hh, bb),
+    h = window.innerHeight,
+    // space between scroll position and top of the footer
+    whenAtBottom = f - t,
+    mh = Math.min(h, whenAtBottom) - hh;
+  $("#sidenav").css({ top: headerHeight, maxHeight: mh });
+  $("#site-toc--side").css({ top: headerHeight, maxHeight: mh });
+}
```

Comment:
What changes did you make to this file? There's a lot of changed lines due to the functions being moved around.

Reply:
Hey @johnpryan. I felt like it was a good time to clean this file up and make it have a consistent style, something we left out of the last cleanup. I did the following:

- I consolidated two document ready jQuery functions into 1 i.e. `$(function() { ... })`
- I moved all inner functions outside of that scope, like the majority of them
- I moved loose code into the ready function
- I moved 2 vars inside the only function that uses them
- I put the jQuery ready function at the bottom of the file
- Added the cookie function

I did not change any logic inside existing functions.

Reply:
Thanks, LGTM

Reply:
Thanks for the clarification and the cleanup!